base/rico-core/src/main/java/dev/rico/internal/core/http/ByteUnit.java

```diff
@@ -0,0 +1,69 @@
+package dev.rico.internal.core.http;
+
+import java.util.Objects;
+
+public enum ByteUnit {
+
+    BYTE("B", "B", "byte", 0),
+    KILOBYTE("KB", "KiB", "kilobyte", 1),
+    MEGABYTE("MB", "MiB", "megabyte", 2),
+    GIGABYTE("GB", "GiB", "gigabyte", 3),
```

Comment:
I think we should not combine this - a Gigabyte is not a Gibibyte. 
Thus there is no default name, it's decimalName or binaryName.